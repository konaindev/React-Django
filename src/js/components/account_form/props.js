export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
