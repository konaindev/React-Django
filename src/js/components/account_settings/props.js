const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
