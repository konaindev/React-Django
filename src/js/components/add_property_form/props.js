export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
