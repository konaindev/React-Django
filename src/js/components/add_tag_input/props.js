const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
