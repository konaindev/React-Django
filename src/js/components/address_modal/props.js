export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
