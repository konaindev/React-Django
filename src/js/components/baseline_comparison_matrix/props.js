import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
