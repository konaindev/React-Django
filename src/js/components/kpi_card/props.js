export const offTrack = {
  health: 0,
  name: "TOU -> APP",
  target: "66%",
  value: "33%"
};

export const atRisk = {
  health: 1,
  name: "INQ -> TOU",
  target: "40%",
  value: "23%"
};

export const onTrack = {
  health: 2,
  name: "USV -> INQ",
  target: "6%",
  value: "5%"
};
