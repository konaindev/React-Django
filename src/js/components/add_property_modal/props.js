import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
