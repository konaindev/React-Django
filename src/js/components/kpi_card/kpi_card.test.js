import renderer from "react-test-renderer";

import KPICard from "./index";
import { offTrack, atRisk, onTrack } from "./props";

describe("SectionHeader", () => {
  it("renders Off Track", () => {
    const tree = renderer.create(<KPICard {...offTrack} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders At Risk", () => {
    const tree = renderer.create(<KPICard {...atRisk} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders On Track", () => {
    const tree = renderer.create(<KPICard {...onTrack} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
