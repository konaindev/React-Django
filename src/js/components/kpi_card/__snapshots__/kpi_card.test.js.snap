// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader renders At Risk 1`] = `
<div
  className="kpi-card kpi-card--at-risk"
>
  <div
    className="kpi-card__bar"
  />
  <div
    className="kpi-card__health"
  >
    <div
      className="property-status property-status--at-risk"
    >
      AT RISK
    </div>
  </div>
  <div
    className="kpi-card__value"
  >
    23%
  </div>
  <div
    className="kpi-card__name"
  >
    INQ -&gt; TOU
  </div>
  <div
    className="kpi-card__target"
  >
    Target: 
    40%
  </div>
</div>
`;

exports[`SectionHeader renders Off Track 1`] = `
<div
  className="kpi-card kpi-card--off-track"
>
  <div
    className="kpi-card__bar"
  />
  <div
    className="kpi-card__health"
  >
    <div
      className="property-status property-status--off-track"
    >
      OFF TRACK
    </div>
  </div>
  <div
    className="kpi-card__value"
  >
    33%
  </div>
  <div
    className="kpi-card__name"
  >
    TOU -&gt; APP
  </div>
  <div
    className="kpi-card__target"
  >
    Target: 
    66%
  </div>
</div>
`;

exports[`SectionHeader renders On Track 1`] = `
<div
  className="kpi-card kpi-card--on-track"
>
  <div
    className="kpi-card__bar"
  />
  <div
    className="kpi-card__health"
  >
    <div
      className="property-status property-status--on-track"
    >
      ON TRACK
    </div>
  </div>
  <div
    className="kpi-card__value"
  >
    5%
  </div>
  <div
    className="kpi-card__name"
  >
    USV -&gt; INQ
  </div>
  <div
    className="kpi-card__target"
  >
    Target: 
    6%
  </div>
</div>
`;
